guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -0,0 +1,5 @@
+We'd like to thank the following people who contributed to the Foreman {ProjectVersion} release:
+
+Adam Růžička,Archana Kumari,Bastian Schmidt,Dirk Götz,Eric Helms,Erik Berg,Evgeni Golov,Ewoud Kohl van Wijngaarden,Gaurav Talreja,Girija Soni,Gordon Bleux,Griffin Sullivan,Griffin-Sullivan,Ian Ballou,Jan Bundesmann,Karolina Malyjurkova,Leos Stejskal,Maria Agaphontzev,Markus Bucher,Markus Reisner,Martin Alfke,Maximilian Kolb,Michal Barecki,Nofar Alfassi,Oleh Fedorenko,Ottavia Balducci,Pat Riehecky,Quinn James,Ron Lavi,Samir Jha,Tim Meusel,William Bradford Clark
```

Comment:
I thought we had one per line. Is the script broken?

Reply:
I see there's an option for `--multi-line` so maybe should try that